app/js/controllers/events.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+angular
+.module('app.controllers')
+.controller('EventsCtrl', function($scope, $location, Restangular) {
+	$scope.loadingPromise = Restangular.one('events/next-10-publicly')
```

Comment:
Well alright then! That's a nifty bit of syntax, I had no idea it existed. Sounds like the way to go.